fluent.syntax/docs/reference.rst

```diff
@@ -0,0 +1,20 @@
+API Reference
+=============
+
+The :py:mod:`fluent.syntax` module contains two helper methods.
+
+The :py:mod:`fluent.syntax.parser` and :py:mod:`fluent.syntax.serializer` modules
+provide more fine-grained control and detail.
+
+
+.. automodule:: fluent.syntax
+   :members:
+
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   parsing
+   ast
+   serializing
```

Comment:
Non-blocking suggestion: I'd make `ast` either first or last in the list.

Reply:
Per matrix, we'll take another look at the ordering when we move the Visitor and Transformer.